tests/js/spec/views/projectDetail/projectFilters.spec.jsx

```diff
@@ -13,46 +13,34 @@ describe('ProjectDetail > ProjectFilters', () => {
 
   it('recommends semver search tag', async () => {
     const organization = TestStubs.Organization();
-    tagValueLoader.mockImplementationOnce(() =>
-      Promise.resolve([
-        {
-          count: null,
-          firstSeen: null,
-          key: 'release.version',
-          lastSeen: null,
-          name: 'sentry@0.5.3',
-          value: 'sentry@0.5.3',
-        },
-      ])
-    );
-    const wrapper = mountWithTheme(
+    tagValueLoader.mockResolvedValue([
+      {
+        count: null,
+        firstSeen: null,
+        key: 'release.version',
+        lastSeen: null,
+        name: 'sentry@0.5.3',
+        value: 'sentry@0.5.3',
+      },
+    ]);
+    render(
       <OrganizationContext.Provider value={organization}>
```

Comment:
You shouldn't need <OrganizationContext.Provider if you pass organization with the context